pkg/userdata/centos/userdata.go

```diff
@@ -190,10 +194,12 @@ write_files:
       socat \
       wget \
       curl \
-      ipvsadm
+      ipvsadm \
+      open-vm-tools
```

Comment:
Should we only install it when we use vSphere as provider?
We have the cloud provider during templating available at `{{ .CloudProvider }}`.

So maybe doing `{{ if eq .CloudProvider "vsphere" }}` could do the job.
Would apply for Ubuntu as well.

In case you're up for a small cosmetic improvement ;)
Otherwise i'll issue a follow up PR